src/debugging/renderdoc.cpp

```diff
@@ -3,22 +3,22 @@
 #include "../platform.hpp"
 #include "../util/logger.hpp"
 
-#if NOVA_WINDOWS
-#include "../util/windows.hpp"
-#include "../util/windows_utils.hpp"
+#if defined(NOVA_WINDOWS)
+#    include "../util/windows.hpp"
```

Comment:
I don't know how I missed this the first time I reviewed but can we not separate the # and the token? Like, I have never done this in my code and I don't know why clang-format thought it was okay